NOTICE

```diff
@@ -47,6 +44,106 @@ types-urllib3 1.26.15 : Apache License 2.0
 urllib3 1.26.9 : MIT License
 zipp 3.6.0 : MIT License
 
+
+Copyright Text: 
+
+Click - Python Command Line Utility 8.0.4 pypi:click/8.0.4
+	Copyright 2002-2006 Python Software Foundation. All rights reserved
+	Copyright 2001-2006 Gregory P. Ward
+	Copyright 2001-2006 Gregory P. Ward. All rights reserved
+	Copyright 2002-2006 Python Software Foundationimport typing as tfrom collections import dequefrom gettext import gettext as _from gettext import ngettext
+CommonMark-py 0.9.1 pypi:commonmark/0.9.1
+	Copyright (c) 2014, John MacFarlaneAll rights reserved
+	Copyright (c) 2014, Bibek Kafle and Roland Shoemaker
+Packaging 21.3 pypi:packaging/21.3
+	copyright = about

```

Comment:
Not right; but somebody has copyright?

Reply:
I added the author noted on PyPI. Good? https://pypi.org/project/packaging/

Reply:
Yes, added that.